spec/BehaviorRelay.spec.ts

```diff
@@ -186,4 +186,43 @@ describe('BehaviorRelay', () => {
     expect(expected).toEqual([]);
     expect(relay.complete).toBeCalled();
   });
+
+  it('should should not emit anything if no event has been pushed', () => {
+    var relay = new BehaviorRelay();
```

Comment:
I think it should be `const`, not `var` :D

Reply:
nice, thank you! 😃